src/opm/input/eclipse/EclipseState/Runspec.cpp

```diff
@@ -607,8 +607,11 @@ Runspec::Runspec( const Deck& deck )
                 std::string msg = "The CO2 storage option is given. PVT properties from the Brine-CO2 system is used \n"
                                   "See the OPM manual for details on the used models.";
                 OpmLog::note(msg);
+            } else if (runspecSection.hasKeyword<ParserKeywords::GASWAT>()) {
+                // TODO: Make sure gas and water phases are actived internally
+                throw std::runtime_error("The CO2 storage option is given with GASWAT activated. Activate GAS and WATER instead.");
```

Comment:
Why are we throwing here? Isn't the point of the code above that we should accept GASWAT + CO2STORE. as equivalent with GAS, WATER + CO2STORE.

Reply:
Yes, I will delete it once the TODO is done (currently working on it).